test/src/test/java/ai/timefold/solver/test/api/score/stream/testdata/justification/TestSecondComparableJustification.java

```diff
@@ -0,0 +1,25 @@
+package ai.timefold.solver.test.api.score.stream.testdata.justification;
+
+import ai.timefold.solver.core.api.score.stream.ConstraintJustification;
+
+public class TestSecondComparableJustification
```

Comment:
Any particular reason for testing comparable justifications? What specifically is being tested here?

Reply:
Yes, the goal is to ensure the logic works for `equals` and `compareTo` methods. We are specifically testing `assertEquality(Object target, Collection<?> otherList) `.

Reply:
If you think there are too many duplicated tests, I'm okay with keeping only 'justifiesComparable()', since the coverage won't be affected.